lib/generators/tenant_level_security/install_generator.rb

```diff
@@ -0,0 +1,11 @@
+module TenantLevelSecurity
+  module Generators
+    class InstallGenerator < ::Rails::Generators::Base
```

Comment:
This gem does not have an explicit dependency on `Rails`, so this reference is not always safe.
It might be better to make it work only if `Rails` is defined.

Strictly speaking, the classes and methods available will likely change depending on the version, so some version constraints may be necessary.